hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -320,18 +338,13 @@ def test_container(container_type: Type[LightningContainer], use_gpu: bool) -> N
     if use_gpu:
         module.cuda()
 
-    assert_train_step(module, data_module, use_gpu)
-    assert_validation_step(module, data_module, use_gpu)
-    assert_test_step(module, data_module, use_gpu)
+    assert_train_step(module, data_module, use_gpu)  # type: ignore
```

Comment:
Why are we ignoring types here? Would this be fixed by narrowing the type annotation for `container_type` (e.g. `Type[BaseMILTiles]`)? Or at least declaring `module: BaseDeepMILModule`?

Reply:
Because this test is in `test_deepmil.py`, I'd think it's safe to assume we'll only test `BaseMIL` containers. We can always generalise the test later if necessary.